azure-pipelines.yml

```diff
@@ -1,19 +1,15 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
 - master
 
-pool:
-  vmImage: 'Ubuntu-16.04'
-
-steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
-
-- script: |
-    echo Add other tasks to build, test, and deploy your project.
-    echo See https://aka.ms/yaml
-  displayName: 'Run a multi-line script'
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: tomtom-international/azure-pipeline-templates
+      ref: refs/heads/feature/python-pipeline
```

Comment:
This will be updated to master once the other PR gets merged.

But we need to think about how we want to test and release the Azure templates and create tags for each version so that we start pinning to specific versions